cluster/gce/trusty/configure.sh

```diff
@@ -0,0 +1,221 @@
+#!/bin/bash
```

Comment:
Good idea. These functions are also useful for coreos, and others are not suitable.
- config_hostname()
- create_dirs()
- create_kubelet_kubeconfig()
- create_kubeproxy_kubeconfig()
- install_kube_binary_config()

Maybe we can refactor the code step by step: (1) move the code from trusty/node.yaml to a script (as shown in this PR); (2) refactor coreos config to make use of the script.